server/src/apub/extensions/signatures.rs

```diff
@@ -2,6 +2,7 @@ use crate::{apub::ActorType, LemmyError};
 use activitystreams_ext::UnparsedExtension;
 use activitystreams_new::unparsed::UnparsedMutExt;
 use actix_web::{client::ClientRequest, HttpRequest};
+use anyhow::{anyhow as format_err};
```

Comment:
Could you change these to just `anyhow::anyhow`, and change all the `format_err!` to `anyhow!` below? thanks.